### PR TITLE
#160986223 Set up Repo to use Pipenv

### PR DESCRIPTION
**Pull request by JoanNgatia**
#### What does this PR do?
Updates the repo to use pipenv to replace the need for pip as well as virtualenv for better dependency management.

#### Description of Tasks to be completed:
- Update pip commands to pipenv.
- Update README to depict changes to a fresh installation on a development environment.
- Update bash script for the installation.

#### How should this be manually tested?
Set up the app on fresh local development environment.

#### Any background context you want to provide?
There have been challenges with tracking updates to package dependencies. Pipenv better handles any dependencies as well as devDependencies.

#### What are the relevant pivotal tracker stories?
#2146417

### CODE REVIEW

File: install_art.sh [hunk 1]
```diff
@@ -43,6 +37,14 @@ create_postgres_instance()
     psql --command "GRANT ALL PRIVILEGES ON DATABASE "art_postgres" to art_postgres;" -U postgres
 }
 
+install_art_requirements()
+{
+    pipenv install
+
+    pipenv run shell
```
Comment (egichuri) [inline install_art.sh]: `pipenv shell`

**Comment by egichuri** [comment]
@JoanNgatia looks like the tests are failing coz of https://github.com/AndelaOSP/art-backend/blob/1507f0c89b5077a6cd18c785b00ae236cd9622d2/.circleci/config.yml#L33

we'll also need to add flake8 (dev requirement)

File: install_art.sh (hunk 1 above)
Comment (JoanNgatia) [reply install_art.sh]: Edited the pipenv command

**Comment by JoanNgatia** [comment]
@egichuri @BrianOwino PR for your review.

**Pull request merged**